staff/YaninaGomezCucchiararo/css-layout-simple-adaptive/style/main-desktop-css.css

```diff
@@ -0,0 +1,54 @@
+html, body {
+    min-height:100% !important;
+    width: 100%;
+    height: 100%;
+}
+
+*{
```

Comment:
TIP many of these css rules are repeated in both mobile and desktop. in that case, is convenient to concentrate them in a single file, like main.css. move them there.